### PR TITLE
End-to-end example for DL training and inference using Spark 3.4 APIs

### PR DESCRIPTION
**Pull request by leewyang**

### CODE REVIEW

**Comment by nvliyuan** [comment]
Hi @leewyang , thanks for the contribution, will verify on databricks

**Comment by leewyang** [comment]
cc @anfeng @eordentlich

File: examples/ML+DL-Examples/Spark-DL/train_inference_e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  databricks secrets put --scope aws-s3 --key aws-secret-key
+  ```
+  **NOTE**: Do not hard-code the values of the secrets anywhere and avoid accidental logging of the values.
+
+- Copy the [init.sh](./init.sh) script to DBFS using the
+  ```
+  databricks fs cp init.sh dbfs:/path/to/init.sh
+  ```
+- Create a multi-node Databricks cluster with the following configuration:
+  - Databricks Runtime Versions: `13.0 ML GPU Runtime`
```
Comment (eordentlich) [inline examples/ML+DL-Examples/Spark-DL/train_inference_e2e/README.md:23]: Looks like 13.1 ML Beta GPU has the patch to predict_batch_udf .

**Review by eordentlich — APPROVED** [review]
👍

**Comment by leewyang** [comment]
Failing markdown link check appears to be in unrelated files.

**Review by eordentlich — APPROVED** [review]
👍

**Pull request merged**